src/components/molecules/Steps/Steps.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import Steps from './Steps'
+
+const steps = ['Step One', 'Step Two', 'Step Three', 'Step Four']
+
+describe('<Steps />', () => {
+  it('renders without crashing', () => {
+    expect(() => shallow(<Steps />)).not.toThrow()
+  })
+
+  it('accepts any extra classes', () => {
+    expect(
+      shallow(<Steps steps={steps} className="my-steps" />).hasClass(
+        'my-steps',
+      ),
+    ).toBe(true)
+  })
+
+  it('renders the step item with class', () => {
+    const renderedComponent = mount(<Steps steps={steps} />)
+    expect(renderedComponent.find('.step').length).toBe(steps.length)
+  })
+})
```

Comment:
@Rajmohanrk We need at least one test that tests passing in the currentSelectedStep and making sure it activates the correct step.

Reply:
@websmith sure, will update the test case for it.